esp/esp/program/modules/migrations/0013_auto_20160709_2038.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('modules', '0012_auto_20160212_1903'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
If by accident you mean "we accidentally left it out of #2049", then yes. I left it here on purpose though :stuck_out_tongue: